packages/arb-token-bridge-ui/src/tokenLists.tsx

```diff
@@ -0,0 +1,46 @@
+export interface BridgeTokenList {
+  id: number
+  originChainID: string
+  url: string
+  name: string
+  isDefault: boolean
```

Comment:
what does isDefault mean here? doesn't seem to be used anywhere

Reply:
oopz